configs/components/augeas.rb

```diff
@@ -89,6 +86,16 @@
     ["./configure --prefix=#{settings[:prefix]} #{settings[:host]}"]
   end
 
+  if platform.name =~ /solaris-10-sparc/
+    # Augeas builds its own gnulib during configure -
+    # This patch to gnulib fixes a linking error around symbol versioning in pthread
+    pkg.add_source 'file://resources/patches/augeas/augeas-1.10.1-gnulib-pthread-in-use.patch'
```

Comment:
Why can't we just use the `apply_patch` method here?

Reply:
gnulib comes from a submodule that's cloned during the configure step, so the files don't exist right away - I'm not aware of a way to accomplish this with vanagon

Reply:
@branan oh, or would it work if I just called `pkg.apply_patch` within the configure block here? I didn't try that.

Reply:
That sounds hacky. I'm fine just doing it manually, but would like a comment better explaining what's up